### PR TITLE
experiments with alexnet - random weights, cifar100, imagenette

### PR DESCRIPTION
**Pull request by imtezcan**
- Train Alexnet implementation on cifar100 & imagenette datasets
- Run pipeline for Alexnet with random, cifar100 and imagenette weights

Changes:
- Create Alexnet without output layer or with sigmoid or softmax outputs
- Experiments on separate notebooks with results
- Delete predictions and correlations from memory after running the pipeline, since we will have already saved them on disk

### CODE REVIEW

File: src/algonauts/models/model_loaders.py [hunk 1]
```diff
@@ -9,6 +10,15 @@ def load_vgg16():
...
+    optimizer = tf.keras.optimizers.Adam()
+    loss = tf.keras.losses.BinaryCrossentropy()
+    model.compile(optimizer=optimizer, loss=loss, metrics='accuracy')
+    transform_image = image_transforms.transform_alexnet
+    return model, transform_image
+
+
 def load_from_file(model_filename, transform_image):
-    model = tf.keras.models.load_model(model_filename, custom_objects={'F1Score': tfa.metrics.F1Score})
+    model = tf.keras.models.load_model(model_filename)  # , custom_objects={'F1Score': tfa.metrics.F1Score})
```
Comment (Efefefef) [inline src/algonauts/models/model_loaders.py:23]: it produces an error if tries to load a model with an unrecognised customisations, so you need to pass it in case you used it to train it

Comment (imtezcan) [reply src/algonauts/models/model_loaders.py:23]: I acutally removed F1Score because I cannot install tensorflow-addons on my macbook. There is no version compatible with M1 tensorflow dependencies. So I removed F1Score when running it locally.

I should add it back for the code though.
